akd_mysql/src/mysql.rs

```diff
@@ -169,7 +173,9 @@ impl<'a> AsyncMySqlDatabase {
             trans: LocalTransaction::new(),
 
             num_reads: Arc::new(tokio::sync::RwLock::new(0)),
+            read_call_stats: Arc::new(tokio::sync::RwLock::new(HashMap::new())),
```

Comment:
Can we make these optional? Or enabled with a compilation feature flag? This is unbounded memory should anyone not flush the call stats. Ints and durations I'm not so worried about, but this is a list that will grow indefinitely. While useful for debugging, it's a mem leak in prod

Reply:
As is, this will not use unbounded memory. Max number of elements is ` < f * t` where `f` is the number of functions we have in the database and `t` is the type of data loaded through these functions.

Below `get_direct` loads three types of data: `Azks`, `HistoryNodeState` and `HistoryTreeNode`. `get_user_state` doesn't have an associated data type, so it can only have one element at max.

`[("get_direct:~Azks", 1), ("get_direct:~HistoryNodeState", 459), ("get_direct:~HistoryTreeNode", 459), ("get_user_state~", 50)]`

Also, if a function is not called, it is not added to the stats.

I think additional burden of this computation and memory-wise is minimal but let me know, I can look into feature flags.

Reply:
OK yeah I see now, you're keying on the string so there is a finite upper limit. OK I'm fine with that then, I was more worried on dynamic key generation for example, with no pruning. But in this case we're safe.